Core/Core/Planner/CalendarToDo/ViewModel/CalendarToDoDetailsViewModel.swift

```diff
@@ -20,19 +20,50 @@ import Foundation
 import Combine
 
 public class CalendarToDoDetailsViewModel: ObservableObject {
+
+    // MARK: - Output
+
     public let navigationTitle = String(localized: "To Do", bundle: .core)
     public let screenConfig = InstUI.BaseScreenConfig(refreshable: false)
 
+    @Published private(set) var state: InstUI.ScreenState = .data
     @Published public private(set) var title: String?
     @Published public private(set) var date: String?
     @Published public private(set) var description: String?
     @Published public private(set) var navBarColor: UIColor?
+    @Published public var shouldShowDeleteConfirmation: Bool = false
+    @Published public var shouldShowDeleteError: Bool = false
```

Comment:
I know, but I prefer to be consistent with other properties and be explicit with type names even if it's not necessary because of the initial value (like Bool, Int, Double, CGFloat, String, etc)
(with the exception of tests and maybe static let constants)